test/WormholeReceiver.t.sol

```diff
@@ -8,7 +8,7 @@ import {WormholeReceiver} from "src/WormholeReceiver.sol";
 import {Constants} from "test/Constants.sol";
 
 contract WormholeReceiverTestHarness is WormholeReceiver {
-  constructor(address _relayer) WormholeBase(_relayer) {}
+  constructor(address _relayer) WormholeBase(_relayer) WormholeReceiver(msg.sender) {}
```

Comment:
I think it would be a little more robust to:

* Accept an `_owner` argument in the harness constructor
* Pass it forward in two the receiver constructor
* Define an `owner` storage var in the `WormholeReceiverTest` contract and pass it forward in `setup`
* Add a test to assert the `owner` is set up correctly
* Prank the `owner` in the `SetRegisteredSender` test
* Add a test that ensures a non-owner address cannot update the registered sender (expect revert)

what do you think

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/55/commits/dd6134f44b596ad6006e7e162a9c7e48defbe410

Reply:
Looks good, thanks @alexkeating